File: 2st/Yeonwoo/수 조작하기1.py
def solution(n, control):
    answer = 0
    for i in control:
        if i == "w":
            n+=1
        elif i == "s":
            n-=1
        elif i == "d":
            n+=10
        elif i == "a":
            n-=10
    answer = n
    return answer

File: 4st/Gau/programmer/수 조작하기 1.java
class Solution {
    public int solution(int n, String control) {
        int answer = n;
        String[] controlList = control.split("");
        for(int i = 0; i<controlList.length; i++){
            if(controlList[i].equals("w")){
                answer+=1;
            }
            else if(controlList[i].equals("s")){
                answer-=1;
            }
            else if(controlList[i].equals("d")){
                answer+=10;
            }
            else if(controlList[i].equals("a")){
                answer-=10;
            }
        }
        return answer;
    }
}
